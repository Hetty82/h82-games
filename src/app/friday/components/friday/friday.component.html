<app-fr-games></app-fr-games>

<br>
<hr>
<br>

<app-fr-active-game></app-fr-active-game>
