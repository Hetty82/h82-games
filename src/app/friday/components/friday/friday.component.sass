@import '~styles/variables'

.games
  display: block
  border-bottom: $border-light
