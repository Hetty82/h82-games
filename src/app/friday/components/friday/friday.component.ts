import { Component, OnInit } from '@angular/core'


@Component({
  selector: 'app-fr-friday',
  templateUrl: './friday.component.html',
  styleUrls: ['./friday.component.sass'],
})
export class FridayComponent implements OnInit {
  constructor() { }

  ngOnInit() {
  }
}
