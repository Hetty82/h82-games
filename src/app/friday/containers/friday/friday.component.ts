import { Component, OnInit } from '@angular/core'


@Component({
  selector: 'app-friday',
  templateUrl: './friday.component.html',
  styleUrls: ['./friday.component.sass']
})
export class FridayComponent implements OnInit {

  constructor() { }

  ngOnInit() {
  }

}
