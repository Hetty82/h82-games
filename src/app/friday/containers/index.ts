import { ActiveGameComponent } from './active-game/active-game.component'
import { GamesComponent } from './games/games.component'

export const containers: any[] = [
  ActiveGameComponent,
  GamesComponent,
]

export * from './active-game/active-game.component'
export * from './games/games.component'
