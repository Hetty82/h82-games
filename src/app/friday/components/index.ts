import { FridayComponent } from './friday/friday.component'

export const components: any[] = [
  FridayComponent,
]

export * from './friday/friday.component'
