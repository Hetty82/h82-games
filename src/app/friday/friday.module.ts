import { NgModule } from '@angular/core'
import { CommonModule } from '@angular/common'
import { RouterModule } from '@angular/router'

import { StoreModule } from '@ngrx/store'

import { routes } from './friday.routes'

import * as fromComponents from './components'
import * as fromContainers from './containers'
import * as fromStore from './store'


@NgModule({
  imports: [
    CommonModule,
    RouterModule.forChild(routes),
    StoreModule.forFeature('friday', fromStore.reducers ),
  ],
  declarations: [
    ...fromComponents.components,
    ...fromContainers.containers,
  ],
  exports: [
    fromComponents.FridayComponent,
  ],
})
export class FridayModule { }
