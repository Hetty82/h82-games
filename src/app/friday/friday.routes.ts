import { Routes } from '@angular/router'

import * as fromContainers from './containers'
import * as fromComponents from './components'

// tslint:disable:object-literal-sort-keys

export const routes: Routes = [
  {
    path: '',
    component: fromComponents.FridayComponent,
    children: [
      {
        path: ':gameId',
        component: fromContainers.ActiveGameComponent,
      },
    ],
  },
]
