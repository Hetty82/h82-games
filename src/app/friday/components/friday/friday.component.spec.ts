import { NO_ERRORS_SCHEMA } from '@angular/core'
import { async, ComponentFixture, TestBed } from '@angular/core/testing'

import { FridayComponent } from './friday.component'


describe('FridayComponent', () => {
  let component: FridayComponent
  let fixture: ComponentFixture<FridayComponent>

  beforeEach(async(() => {
    TestBed.configureCompiler({ preserveWhitespaces: false } as any).configureTestingModule({
      declarations: [ FridayComponent ],
      schemas: [ NO_ERRORS_SCHEMA ],
    })
    .compileComponents()
  }))

  beforeEach(() => {
    fixture = TestBed.createComponent(FridayComponent)
    component = fixture.componentInstance
    fixture.detectChanges()
  })

  it('should create component', () => {
    expect(component).toBeTruthy()
  })

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot()
  })
})
