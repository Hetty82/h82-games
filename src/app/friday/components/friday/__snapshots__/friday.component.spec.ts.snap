// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FridayComponent should match snapshot 1`] = `
<app-fr-friday>
  <app-fr-games /><br /><hr /><br /><app-fr-active-game />
</app-fr-friday>
`;
